components/homepage/PanelSix.vue

```diff
@@ -103,30 +103,33 @@
             md:px-8
           "
         >
-          <AdvancedThumbnailCard
-            title="MAU Part 1 Intro"
-            :is-large="true"
-            author="Admin"
-            date="17/10/2020"
-            content=" Lorem ipsum dolor sit amet, consectetur adipisicing elit. Voluptatibus quia, nulla! Maiores et perferendis eaque, exercitationem praesentium nihil."
-          />
-          <AdvancedThumbnailCard
-            title="MAU Part 1 Intro"
-            :is-large="true"
-            author="Admin"
-            date="17/10/2020"
-            content=" Lorem ipsum dolor sit amet, consectetur adipisicing elit. Voluptatibus quia, nulla! Maiores et perferendis eaque, exercitationem praesentium nihil."
-          />
-          <AdvancedThumbnailCard
-            title="MAU Part 1 Intro"
-            :is-large="true"
-            author="Admin"
-            date="17/10/2020"
-            content=" Lorem ipsum dolor sit amet, consectetur adipisicing elit. Voluptatibus quia, nulla! Maiores et perferendis eaque, exercitationem praesentium nihil."
-          />
+          <div v-for="post of posts" :key="post.slug">
+            <AdvancedThumbnailCard
+              :title="post.title"
+              :is-large="true"
+              :author="post.author"
+              :date="post.date"
+              :content="post"
```

Comment:
Let's use the 'Excerpt' function in Nuxt content here. See: https://content.nuxtjs.org/displaying#excerpt

and https://content.nuxtjs.org/displaying#excerpt